q2_types/per_sample_sequences/_transformer.py

```diff
@@ -94,6 +94,12 @@ def _11(dirfmt: CasavaOneEightLanelessPerSampleDirFmt) \
         dirfmt, SingleLanePerSamplePairedEndFastqDirFmt, parse_lane=False)
 
 
+@plugin.register_transformer
+def _12(dirfmt: SingleLanePerSampleSingleEndFastqDirFmt) \
+        -> CasavaOneEightSingleLanePerSampleDirFmt:
+    return CasavaOneEightSingleLanePerSampleDirFmt(str(dirfmt), mode='r')
```

Comment:
I am 90% sure this is still retaining the `metadata.yml` and `manifest` files, which we wanted to ditch in this transformer. However... your test isn't failing, and I don't know entirely what to make of that without looking closer.

Reply:
Hmm, I'll take a look at that

Reply:
@ebolyen the post transformation directory contains a single `.fastq.gz` containing one `.fastq` file. That's all

Reply:
Well that's certainly the goal. So if you apply this to a typical `demux.qza` file, you see similarly only fastq.gz files?

Reply:
No, it looks like that does break

![IPython: home-anthony_091](https://user-images.githubusercontent.com/10642616/64033812-27d2a480-cb02-11e9-833c-a5ce4a4c85fd.png)

Reply:
Ok! that makes sense then. So we must have some less-than-correct test data which is why this is passing.

Reply:
Yeah, I looked at the test again (I mostly copy pasted the test from the casava to slpssefdf) and it only uses a .fastg.gz as input. I'll add some new test data with a manifest and a metadata then make the test pass.